src/ProjectTemplates/Web.ProjectTemplates/content/EmptyComponentsWebAssembly-CSharp/Client/Properties/launchSettings.json

```
@@ -0,0 +1,46 @@
+{
+  "iisSettings": {
+    "iisExpress": {
+      "applicationUrl": "http://localhost:8080",
+      //#if(HasHttpsProfile)
+      "sslPort": 44300,
```

Looks like the CI isn't happy with this trailing comma.

```suggestion
      "sslPort": 44300
```

https://dev.azure.com/dnceng/public/_build/results?buildId=1831669&view=ms.vss-test-web.build-test-results-tab&runId=48473968&resultId=122793&paneView=debug